server/route.go

```diff
@@ -76,6 +76,8 @@ func buildRoutes(app *applicationHandler) http.Handler {
 			r.Use(jsonResponse)
 			r.Use(requireAuth())
 
+			r.Post("/process_organisation_member_invite", app.ProcessOrganisationMemberInvite)
```

Comment:
Not sure this route follows the `REST` pattern.

Reply:
True, we'd need to create an invites resource for that then

Reply:
`/organisations/members/accept-invite` as an example and it can be under the `/ui` route.